app/services/ai_chat_engine.py

```
@@ -1357,7 +1417,27 @@ async def _handle_rebalancing(self, message: str, context: Dict, user_id: str) -
+            
+            # 2a. For strategy comparison requests, show all 6 strategies
+            if wants_comparison or strategy == "auto":
+                try:
+                    strategy_comparison = await self.chat_adapters._analyze_all_strategies_comprehensive(
+                        portfolio_data={}, user_id=user_id
+                    )
+                    
+                    if strategy_comparison and strategy_comparison.get("strategy_results"):
+                        return self._format_strategy_comparison_response(strategy_comparison, user_id)
+                    
+                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)